### ISSUE TITLE
Not all Couchpotato wanted movies is being imported

### ISSUE THREAD

**Issue by apperrault** [Radarr/Radarr#783]
**Description:**

Hi all, I have a Couchpotato install that I am looking to move off of.  I have almost everything working the way I want it, but I currently have 44 items on my Couchpotato wanted list, but when I press Import, only some of the movies are moving over.  I can post logs, once i know where to look for them.  Has anyone seen this happen?

**Radarr Version:**
0.2.0.299 - Feb 7 2017  develop (QNAP)
Mono Version 4.6.2 (Stable 4.6.2.7/08fd525 mercredi 23 novembre 2016, 17:45:54 (UTC+0100))
**Logs:**

Please use the search bar and make sure you are not submitting an already submitted issue.
Visit our [Discord server](https://discord.gg/NWYch8M) for support or longer discussions.

**Comment by galli-leo** [comment]
Logs are under System -> Logs. They would be very helpful in this case

**Comment by apperrault** [comment]
Ok. I am at a swim meet all day today but I will grab them this evening and
post them.

**Comment by apperrault** [comment]
[radarr.txt](https://github.com/Radarr/Radarr/files/791960/radarr.txt)
here is the radarr.txt logs from today.  Right before i downloaded this, i went to Movies, and clicked RSSSync.  I have a total of 46 movies in CP that are wanted, but now I only have 21 wanted in Radarr.  It isn't that many so if i have to just re-add them to Radarr I will, but I would love to see this feature working

Aaron

**Comment by galli-leo** [comment]
@apperrault You have to go to Add Movie -> Import From list to import from Couch Potato. Not RssSync. But by the looks of the logs, the connection to CP timed out.

**Comment by apperrault** [comment]
OK, thanks.  So, I did a little more digging and stumbled across what i think to be an error.  One of the movies that I had in my CP wanted list is the new Beauty and the Beast movie coming out next month.  I went into Radarr and tried to add it to the wanted list and it said it already existed.  I was a little confused because it isn't in my wanted list on Radarr.  I did a search of my movies and it indeed does say it exists, but it says it exists at the following path:

/share/CACHEDEV1_DATA/.qpkg/Radarr/Radarr/Select Path/Beauty and the Beast (2017)

that is not where my movies should be living.  it should be at:

/share/CACHEDEV1_DATA/Media/Movies/Beauty and the Beast (2017)

I deleted the bad entry and re-performed an import, and it showed up with the proper location.  

Very odd if you ask me.  should I go through and delete the other bad ones that show up and that way and re-do an import?

app
